main/cui/source/dialogs/about.cxx

```diff
@@ -502,13 +502,14 @@ const rtl::OUString AboutDialog::GetBuildId() const
     // For better readability, strip the duplicate ProductMajor ("340").
     if ( sProductSource.getLength() )
     {
+        sal_Int32 versionLength = sProductSource.getLength() - 3;
```

Comment:
Actually, this should be named "productMajorLength" to be consistent with the above comment. It's probably worth renaming for the sake of clarity.